lib/dictionary_class.rb

```diff
@@ -0,0 +1,12 @@
+module Scrabble
+  class Dictionary
+    def self.word_in_dictionary(guess)
+      open("./lib/dictionary.txt", "r")
+      if File.open("./lib/dictionary.txt").each_line.any?{|line| line.include?(guess)}
```

Comment:
There is way too much logic happening on this single line of code. This should be split out into multiple lines.